src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue18443.cs

```diff
@@ -0,0 +1,22 @@
+﻿#if !MACCATALYST && !IOS
+using NUnit.Framework;
+using UITest.Core;
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue18443 : _IssuesUITest
+	{
+		public Issue18443(TestDevice device) : base(device) { }
+
+		public override string Issue => "[Android] SelectionLength Property Not Applied to Entry at Runtime";
+
+		[Test]
+		[Category(UITestCategories.Entry)]
+		public async Task EntrySelectionLengthRuntimeUpdate()
+		{
+			await Task.Delay(500);
```

Comment:
Can you add an automationID to the entry here?

https://github.com/dotnet/maui/pull/24064/files#diff-554128f8a463e1cf14e7869eb36cfacebc92ab4e0c3418276de6023f213a7d79R6

And then use `WaitForElement` to wait for the element to show up on the screen. Typically we'd like to avoid any `Task.Delays`

Reply:
I have modified the code accordingly.